packages/api/src/hooks/useSettings.ts

```diff
@@ -9,7 +10,8 @@ type TSetSettingsPayload = NonNullable<
 
 /** A custom hook to get and update the user settings. */
 const useSettings = () => {
-    const { data, ...rest } = useQuery('get_settings');
+    const { isSuccess } = useAuthorize();
+    const { data, ...rest } = useQuery('get_settings', { options: { enabled: isSuccess } });
```

Comment:
This logic is already added to master branch. Can you take a pull and check

Reply:
This is resolved after the latest master pull 🙏🏼